### DISCUSSION TITLE
Create a more splitted territory to increase LCZ accuracy for impervious type

### DISCUSSION THREAD

**Discussion by j3r3m1** [orbisgis/geoclimate#834] [Ideas]
In rural area or area nearby cities, TSU are getting bigger and thus the lack of data might lead to big misassignements. Here are few examples:
1. an area in Rezé nearby Nantes where a large TSU is affected to impervious while this imperviousness comes from a very small fraction of territory (ground is set as impervious in two "industrial" zones located on the north west and in the east of the TSU). --> this problem might be reduced using the "highway = pass" tag and also the "waterway = stream" tag
![image](https://github.com/orbisgis/geoclimate/assets/13120405/55505ac3-2f85-434c-b4e8-49f5f98857c3)
![image](https://github.com/orbisgis/geoclimate/assets/13120405/69947458-9c93-4525-bc45-8f6443c13157)
2. same slightly more in the South of Nantes: again using the small paths might solve the problem
![image](https://github.com/orbisgis/geoclimate/assets/13120405/128959f8-9217-4b59-a976-070fb2b2d820)
![image](https://github.com/orbisgis/geoclimate/assets/13120405/2aa774fa-cbd0-40ac-9534-8f139bfdba11)
3. again the same in the south-east of Nantes: but this time there is no paths that might help. An idea could be to create TSU based on surfaced of defined land and include surfaces of undefined land, thus avoiding defined lands to act on undefined lands...
![image](https://github.com/orbisgis/geoclimate/assets/13120405/bf4d7ff7-82c0-44a8-bb72-bfde26805113)
![image](https://github.com/orbisgis/geoclimate/assets/13120405/0ff314c1-a6d7-4961-a64c-4225ae05aae5)

**Comment by j3r3m1** [comment]
OK, there is a need to create a drawing showing all the needed inputs and how to use them in the partitionning (landuse residential but also commercial, industrial, etc. and if there is a condition of size).

**Comment by ebocher** [comment]
Sounds a good idea and way to improve LCZ. We need to check the impact on RSU  on several sample zones.

**Comment by ebocher** [comment]
Here the new approach with Reze (first one)
![reze](https://github.com/orbisgis/geoclimate/assets/1820572/cda7d2b8-754c-48d5-8fe9-aed39c8c4a8e)

EDIT: (before)
![](https://user-images.githubusercontent.com/13120405/268005405-69947458-9c93-4525-bc45-8f6443c13157.png)

**Comment by ebocher** [comment]
By the way we can think also about a stream layer

**Comment by j3r3m1** [comment]
works perfectly. A stream layer or adding streams into the water layer (setting a default width of about 1 m) ?

**Reply by ebocher** [reply]
But what about if the stream is only a "ruisseau"

**Reply by j3r3m1** [reply]
I would say set it to 0.5 by default then =)
It depends the use of this stream layer. Using it only for splitting might be a bit ashame since I supposed we would also like to add it as a water surface at the end

**Comment by ebocher** [comment]
New result for "again the same in the south-east of Nantes: but this time there is no paths that might help. An idea could be to create TSU based on surfaced of defined land and include surfaces of undefined land, thus avoiding defined lands to act on undefined lands..."

![geoms](https://github.com/orbisgis/geoclimate/assets/1820572/309b7bf0-e46e-473f-a893-ecb2ca8820e4)
![image](https://github.com/orbisgis/geoclimate/assets/13120405/20a4f600-214a-4157-b106-0755231d95d3)
